networking/p2p/src/main/java/tech/pegasys/teku/networking/p2p/libp2p/LibP2PPeer.java

```diff
@@ -76,6 +81,11 @@ public PeerAddress getAddress() {
     return peerAddress;
   }
 
+  @Override
+  public Double getGossipScore() {
+    return gossip.getGossipScore(connection.secureSession().getRemoteId());
```

Comment:
We could just save the `peerId` as a private instance variable in the constructor so we don't have to make this complex call. 
```suggestion
    return gossip.getGossipScore(peerId);
```

Reply:
Done. Thank you!